spec/ics-003-connection-semantics/README.md

```diff
@@ -331,31 +328,31 @@ function connOpenTry(
   proofConsensus: CommitmentProof,
   proofHeight: Height,
   consensusHeight: Height) {
-    abortTransactionUnless(validateConnectionIdentifier(desiredIdentifier))
+    if (previousIdentifier !== "") {
+      previous = provableStore.get(connectionPath(identifier))
+      abortTransactionUnless(
+        (previous === null) ||
```

Comment:
```suggestion
        (previous !== null) &&
```
If a previous connection identifier is specified, it is [assumed that it must exist](https://github.com/cosmos/cosmos-sdk/blob/a648325f885cbc6d4a354ffbd3f66b9e5ef23728/x/ibc/core/03-connection/keeper/handshake.go#L82) in the implementation

Reply:
ah yes - thanks!